packages/foam-vscode/src/utils/vsc-utils.ts

```diff
@@ -1,8 +1,17 @@
-import { Position, Range } from 'vscode';
-import { Position as FoamPosition, Range as FoamRange } from 'foam-core';
+import { Position, Range, Uri } from 'vscode';
+import {
+  Position as FoamPosition,
+  Range as FoamRange,
+  URI as FoamURI,
+  uris,
+} from 'foam-core';
 
 export const toVsCodePosition = (p: FoamPosition): Position =>
   new Position(p.line, p.character);
 
 export const toVsCodeRange = (r: FoamRange): Range =>
   new Range(r.start.line, r.start.character, r.end.line, r.end.character);
+
+export const toVsCodeUri = (u: FoamURI): Uri => Uri.parse(uris.toString(u));
```

Comment:
Can we add a test that does `toVsCodeUri(fromVsCodeUri(uri))` and check we don't lose any information (and vice versa)